package.json

```diff
@@ -15,8 +15,15 @@
     ]
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "browserify --entry src/index.js | uglifyjs --output dist/bundle.js",
+    "dev": "watch 'npm run build' src",
+    "test": "npm run -s test:code && npm run -s test:style",
+    "test:code": "tap -R nyan --cov test/**/*.js",
+    "test:these-files": "tap --reporter=spec",
+    "test:style": "standard --verbose | snazzy && htmlhint app index.html",
+    "test:debug": "tap --debug -t 0 test/**/*.js",
+    "test:these": "tap -R nyan",
```

Comment:
test:these and test:these-files do pretty much the same thing just with a different reporter. I'd say keep `test:these` as I like that syntax better.